docs/tutorial.rst

```diff
@@ -1385,6 +1361,37 @@ may have one of the following signatures:
         # examples the return code would be 0/success.
         return (None, "I failed", 2)
 
+        # Anything you print to stdout or stderr
+        # will be captured for you automatically. This allows callable
+        # aliases to support piping.
+        print('I go to stdout and will be printed or piped')
+
+        # Note: that you have access to the xonsh
+        # built-ins if you 'import builtins'.  For example, if you need the
+        # environment, you could do the following:
+        import builtins
+        env = builtins.__xonsh__.env
```

Comment:
I didn't change that line (I just moved it from below). I think it was trying to demo the builtins package, rather than showing how to access `__xonsh__`. I'd prefer to remove it in a separate PR since I don't understand the example deeply.